{
    "checksum": "0b0b4ce12dc76eb600137c81322423879326f37defc045e76ed41e5191e5c40b",
    "file": "RecoverableException.cc",
    "mime": {
        "encoding": "us-ascii",
        "type": "text/x-c"
    },
    "number_of_duplicates": 4,
    "package": "aria2",
    "path": "aria2/1.36.0-1/src/RecoverableException.cc",
    "pkg_infos": {
        "area": "main",
        "copyright": true,
        "ctags_count": 0,
        "license": "/copyright/license/aria2/1.36.0-1/",
        "metric": {
            "size": 20772
        },
        "pts_link": "https://tracker.debian.org/pkg/aria2",
        "sloc": [
            [
                "cpp",
                113789
            ],
            [
                "ansic",
                8755
            ],
            [
                "sh",
                4932
            ],
            [
                "makefile",
                1465
            ],
            [
                "ruby",
                475
            ],
            [
                "python",
                235
            ],
            [
                "xml",
                176
            ],
            [
                "sed",
                16
            ]
        ],
        "suites": [
            "bookworm",
            "sid"
        ],
        "vcs_browser": "https://salsa.debian.org/debian/aria2",
        "vcs_type": "git"
    },
    "raw_url": "/data/main/a/aria2/1.36.0-1/src/RecoverableException.cc",
    "stat": {
        "perms": "rw-r--r--",
        "size": 3287,
        "symlink_dest": null,
        "type": "-"
    },
    "text_file": true,
    "type": "file",
    "version": "1.36.0-1"
}